.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="src" path="core"/>
+	<classpathentry kind="src" path="rtf"/>
+	<classpathentry kind="src" path="rups"/>
+	<classpathentry kind="src" path="toolbox"/>
+	<classpathentry kind="lib" path="C:/Users/g.pazhev/.m2/repository/org/bouncycastle/bcprov-jdk15on/1.68/bcprov-jdk15on-1.68.jar"/>
```

Comment:
can you switch the project to maven container or at least not commit the .project file to avoid internal file names.

Reply:
OK. I will not commit the class-path file.